dependencies/pom.xml

```diff
@@ -110,6 +110,7 @@
         <version.lib.microprofile-reactive-streams-operators-core>1.0.1</version.lib.microprofile-reactive-streams-operators-core>
         <version.lib.microprofile-rest-client>2.0</version.lib.microprofile-rest-client>
         <version.lib.microprofile-tracing>1.3.3</version.lib.microprofile-tracing>
+        <version.lib.microstream>05.00.01-MS-GA</version.lib.microstream>
```

Comment:
Could this be upgraded to the latest version (05.00.02-MS-GA)?

Reply:
An upgrade to 05.00.02-MS-GA is fine, i just did it with commit ce530f4